run.py

```diff
@@ -67,7 +67,6 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10, mod
         model_analyzer = ModelAnalyzer()
         if export_dcgm_metrics_file:
             model_analyzer.set_export_csv_name(export_dcgm_metrics_file)
-            model_analyzer.add_mem_throughput_metrics()
```

Comment:
If we are removing this function call, should we remove this function as well?

Reply:
Yes, we can. I'm thinking about adding an API to let users add other metrics more easily without change TorchBenchAnalyzer.py. 
For now, let's remove this function.

Reply:
solved by https://github.com/pytorch/benchmark/pull/1049/commits/7580a44a4cd51edb3623889513328b0047b4a003